src/lib/datepicker/datepicker.ts

```diff
@@ -49,6 +49,11 @@ let datepickerUid = 0;
   selector: 'md-datepicker-content',
   templateUrl: 'datepicker-content.html',
   styleUrls: ['datepicker-content.css'],
+  host: {
+    '[class.mat-datepicker-content]': 'true',
```

Comment:
You should be able to just do 
```ts
host: {
  'class': 'mat-datepicker-content'
}
```
And Angular will union it with the other classes and not overwrite (so long as it isn't a `'[class']` binding)